### PR TITLE
New feature/collapse

### PR DESCRIPTION
**Pull request by jeanluciano**
# Description
Changed the Incidents page to use panels instead of cards. Furthermore, the filtering options are expanded, and the export button is now moved to this page from about page. The tags are currently in random order but will be sorted by weight in the future. 

Fixes # (issue)

The user experience for searching and exporting incidents.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] `npm test`

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/incidents/IncidentFilter.js [hunk 1]
```diff
@@ -43,6 +43,18 @@ const filterDataByDate = (data, range) => {
...
+    data.forEach(incident => {
+      if (incident.categories.indexOf(tag) > 0) {
+        filtered.push(incident);
+      }
+    });
+  });
+  return filtered;
+};
+
 // in progress
```
Comment (Brandonx1123) [inline src/components/incidents/IncidentFilter.js:58]: maybe delete these notes

File: src/components/incidents/IncidentFilter.js [hunk 2]
```diff
@@ -43,6 +43,18 @@ const filterDataByDate = (data, range) => {
   );
 };
 
+const filterByTags = (data, tags) => {
+  let filtered = [];
```
Comment (Brandonx1123) [inline src/components/incidents/IncidentFilter.js:47]: the logic makes sense here

File: src/components/incidents/Incidents.js [hunk 3]
```diff
@@ -1,16 +1,17 @@
 import React, { useEffect, useState } from 'react';
-import IncidentsCard from '../incidents/IncidentsCard';
```
Comment (Brandonx1123) [inline src/components/incidents/Incidents.js:2]: do we need this component anymore?

**Review by Brandonx1123 — APPROVED** [review]

**Review by stevenpmcfarlane — COMMENTED** [review]
Not much of a description of all the changes, of which there are many. Can you say a bit about what the changes are?

**Review by john-chamberlin — APPROVED** [review]
Looks good on paper

**Pull request merged**